content/blog/2024-01-07-INTA/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Citizen science and governance models
+author: Yanina Bellini Saibene
+summary: " "
+date: '2024-01-07'
+categories:
+  - English
+  - Community
+  - 100DaysToOffload
+tags:
+  - English
+  - Community
+  - 100DaysToOffload
+---
+
+{{< figure src="featured.jpeg" alt="Field day at INTA. INTA's researcher showing soil trials and other related research activities and results">}}
+
+Continuing with the reflections on the governance models of organizations, communities of practice and research organizations, today I want to share with you the governance model of the National Institute of Agricultural Technology (INTA) where I worked as a researcher for 24 and a half years and how this related with Citizen Science.
+
+## INTA 
+
+INTA is an Argentine federal research and extension agency in charge of the generation, adaptation and diffusion of technologies, knowledge and learning procedures for the agriculture, forest and agro-industrial activities within an ecologically clean environment. It was created in 1956 and have 44 experimental agricultural stations and 240 extension units around the country.  It is the research institution with the largest territorial presence in our country. 
+
+In 1986, with the return to democracy in 1983, the president issued Decree 287, which modified the law that created INTA. The most important modifications were the _decentralization of the institute's operations_ and the **participation in its management of the different sectors involved in the generation and adoption of technology in each unit**.
+
+The decree makes it clear that "In the regional operating units, the **balanced participation** of the different sectors of agricultural production, provincial governments, universities and the scientific-technical community **in their management will ensure the correspondence between the activities of the units and local problems.**" 
+
+## Implementation
+
+The INTA has a directive board composed of members of both the public and the private sector and with representation for different actors in the agricultural sector; they define the politics at national level. The national direction executes the planning of the board, assisted by the organization and administration technical areas.
+
+Each Regional and Research Center has a Regional board of director (I was the firs female member of the Natural Resource Research Center representing researcher that work at the experimental stations) that define the strategy ar regional level aligned with the national policy. Finally, the experimental stations and extension agencies have local advisory councils with the representation of local actors of the sector. All these governance body meet every month.  The National and Regional boards participate in the definition of the budget, the long- and medium-term strategic plan and the hiring of personnel. 
```

Comment:
"first" not "firs"

Reply:
"representing researchers" (plural)

Reply:
It took me a while to read the first sentence - can you move the part in parentheses to a second sentence?